org.adempiere.base/src/org/compiere/model/MInOutConfirm.java

```diff
@@ -841,5 +841,18 @@ public int getC_Currency_ID()
 	{
 		return 0;
 	}	//	getC_Currency_ID
-	
+
+	/**
+	 * 	Document Status is Complete, Closed, Reversed or Voided
+	 *	@return true if CO, CL, RE or VO
+	 */
+	public boolean isComplete()
+	{
+		String ds = getDocStatus();
+		return DOCSTATUS_Completed.equals(ds)
+			|| DOCSTATUS_Closed.equals(ds)
+			|| DOCSTATUS_Reversed.equals(ds)
+			|| DOCSTATUS_Voided.equals(ds);

```

Comment:
this is now inconsistent with isComplete() in other model class

Reply:
solved